pkg/ovs/ovsconfig/default_windows.go

```diff
@@ -25,9 +25,9 @@ const (
 
 	defaultConnNetwork = "winpipe"
 	namedPipePrefix    = `\\.\pipe\`
-	// Wait up to 2 seconds when get port, the operation of port creation
+	// Wait up to 3 seconds when get port, the operation of port creation
```

Comment:
Why change the waiting delay?

Reply:
Sorry, it's a mistake. Restore the value in update.